src/net_processing.h

```diff
@@ -143,16 +129,26 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
      */
     void Misbehaving(const NodeId pnode, const int howmuch, const std::string& message);
 
-    /** Get statistics from node state */
-    bool GetNodeStateStats(NodeId nodeid, CNodeStateStats& stats);
-
-    /** Set the best height */
-    void SetBestHeight(int height) { m_best_height = height; };
+    /**
+     * Evict extra outbound peers. If we think our tip may be stale, connect to an extra outbound.
+     * Public for unit testing.
```

Comment:
Could have been better to comment in commit message the re-ordering logic followed "Move method exposed for test-only at the end of public method space declaration" :) ?